carbonmark/pages/retire/index.tsx

```diff
@@ -1,18 +1,28 @@
 import { Retire } from "components/pages/Retire";
+import { getCarbonmarkProject } from "lib/carbonmark";
 import { loadTranslation } from "lib/i18n";
 import { GetStaticProps } from "next";
 
+const featuredProjectKeys = ["VCS-1577-2015", "VCS-1764-2020", "VCS-1121-2018"];
+
 export const getStaticProps: GetStaticProps = async (ctx) => {
   try {
     const translation = await loadTranslation(ctx.locale);
 
+    const featuredProjects = await Promise.all(
+      featuredProjectKeys.map(
+        async (project) => await getCarbonmarkProject(project)
+      )
+    );
```

Comment:
the nittiest of picks. I have to suggest something to prove to you that I reviewed the PR

```suggestion
    const featuredProjects = await Promise.all(
      featuredProjectKeys.map(getCarbonmarkProject)
    );
```